tests/ImageSharp.Benchmarks/Codecs/DecodeTiff.cs

```diff
@@ -1,38 +1,94 @@
 // Copyright (c) Six Labors.
 // Licensed under the Apache License, Version 2.0.
 
+//// #define BIG_TESTS
+
 using System.IO;
+
 using BenchmarkDotNet.Attributes;
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff;
 using SixLabors.ImageSharp.PixelFormats;
 using SixLabors.ImageSharp.Tests;
+
 using SDImage = System.Drawing.Image;
 using SDSize = System.Drawing.Size;
 
 namespace SixLabors.ImageSharp.Benchmarks.Codecs
 {
-    [Config(typeof(Config.ShortClr))]
+    [Config(typeof(Clr))]
     public class DecodeTiff : BenchmarkBase
     {
-        private byte[] tiffBytes;
+        public class Clr : Config.ShortClr
+        {
+            public Clr() => this.SummaryStyle = BenchmarkDotNet.Reports.SummaryStyle.Default.WithMaxParameterColumnWidth(60);
+        }
+
+        private string prevImage = null;
+
+        private byte[] data;
+
+        private Configuration configuration;
+
+#if BIG_TESTS
+        private static readonly int BufferSize = 1024 * 68;
```

Comment:
what is the reason to change the buffer size here?

Reply:
for large files suitable a larger buffer  

quote from spec:
> For maximum compatibility with operating systems such as MS-DOS and Windows, the StripOffsets array should be less than or equal to 64K bytes in length,
> and the strips themselves, in both compressed and uncompressed forms, should
> not be larger than 64K bytes

64+4